packages/manager/apps/pci-kubernetes/src/pages/list/useDatagridColumn.spec.tsx

```diff
@@ -29,4 +35,22 @@ describe('useDatagridColumn', () => {
 
     expect(columns).toHaveLength(columnCount);
   });
+
+  it.each(clusterPlans)('should display expected cluster plan', (plan) => {
+    const { result } = renderHook(() => useDatagridColumn());
+
+    const fakeKube = [{ plan } as TKube];
+
+    const { getByText } = render(
+      <Datagrid
+        columns={result.current}
+        items={fakeKube}
+        totalItems={fakeKube.length}
+      />,
+      { wrapper },
```

Comment:
Since you are testing useDatagridColumn I think you should abstract the datagrid and test directly on the result

Reply:
Not sure I understand what you mean by this, is it possible to test the text cell rendered without giving data to the `Datagrid` component?